### PR TITLE
Language class -> get() functions was removed from controllers [small part]

### PR DESCRIPTION
**Pull request by hkaraoglu**
get() function callings was removed  from Controller classes which uses get function in the Language class .Instead of this, data array variable in Language class, was encapsulated to get it from another class. And this was used to merge with controller data. After merging, no need to call get function of Language class. Notice: This commit just contains small part of refactoring of controllers which use get function of Language class. If this commit is accepted, we will continue to refactor controllers.

### CODE REVIEW

File: upload/catalog/controller/account/account.php [hunk 1]
```diff
@@ -8,6 +8,7 @@ public function index() {
 		}
 
 		$this->load->language('account/account');
+		$this->data = array_merge($this->data, $this->language->getData());
```
Comment (osworx) [inline upload/catalog/controller/account/account.php]: How many times do you want to merge?
Easier and simplier:

> $this->data = $this->load->language('account/account');
if first,
> $this->data += $this->load->language('account/account');
if data already has assigned vars.

Comment (Revenge94) [inline upload/catalog/controller/account/account.php]: Good idea. I didn't realize that language method returns data.

Comment (Revenge94) [inline upload/catalog/controller/account/account.php]: I tried your advice. But it didn't work.

**Comment by Revenge94** [comment]
I updated my code by your advices and load function.

**Pull request closed without merging**